src/collective/cover/static/layout_base.js

```diff
@@ -12,9 +12,12 @@ $(document).ready(function() {
                 return (css.match (/\bbtn-\S+/g) || []).join(' ');
             });
             $this.find('span').text('Saving...');
+            var authenticator = jQuery('form[action$="layoutedit"] input[name="_authenticator"]').val();
```

Comment:
@alecpm this has to be implemented in a way that works even if CSRF protection is not implemented